src/functions-and-arrays.js

```diff
@@ -26,16 +50,38 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(arr) {
+  if (arr.length === 0) return null;
+  let sum = 0;
+  arr.forEach(function(element) {
+    sum = sum + element;
+  } );
+  let average = sum / arr.length;
```

Comment:
Here, can you think of another solution using the previous `function sumNumbers()` ? Instead of calculating sum again ;)

Remember, that `sumNumbers()` returns a sum. And we can use it here